src/main/java/uk/co/ramp/covid/simulation/population/Places.java

```diff
@@ -47,6 +51,18 @@ public Places() {
         shops = new ProbabilityDistribution<>();
         careHomes = new ProbabilityDistribution<>();
         all = new ArrayList<>();
+        createPlaceMap();
+    }
+    
+    private void createPlaceMap() {
+        placeMap = new HashMap<>();
+        placeMap.put("ConstructionSites", () -> getConstructionSites());
+        placeMap.put("Hospitals", () -> getNonCovidHospitals());
+        placeMap.put("COVIDHospitals", () -> getCovidHospitals());
+        placeMap.put("Nurseries", () -> getNurseries());
+        placeMap.put("Restaurants", () -> getRestaurants());
+        placeMap.put("Schools", () -> getSchools());
+        placeMap.put("CareHomes", () -> getCareHomes());
```

Comment:
Offices?

Reply:
Thanks for noticing this - it's actually dead code since a refactoring!